sky/utils/cli_utils/status_utils.py

```diff
@@ -408,4 +408,4 @@ def _get_estimated_cost_for_cost_report(
     if not cost:
         return '-'
 
-    return f'${cost:.3f}'
+    return f'$ {cost:.2f}'
```

Comment:
nit: i would prefer to have both without space between the `$` and the number. (very minor, feel free to ignore).

Reply:
We can do a consistency change later, including with `show-gpus`, optimizer outputs, etc. :)